compliance/fedramp.md

```diff
@@ -0,0 +1,11 @@
+---
+description: fedramp
```

Comment:
These descriptions could use some work. It should be a sentence or two, I think.